src/AzureExtension/DevBox/DevBoxProvider.cs

```diff
@@ -27,37 +34,63 @@ string IComputeSystemProvider.DefaultComputeSystemProperties
 
     public string Properties => throw new NotImplementedException();
 
-    public ComputeSystemProviderOperation SupportedOperations => throw new NotImplementedException();
+    // No create operation supported
+    public ComputeSystemProviderOperation SupportedOperations => 0x0;
 
     private bool IsValid(JsonElement jsonElement)
     {
         return jsonElement.ValueKind != JsonValueKind.Undefined;
     }
 
-    public IEnumerable<IComputeSystem> GetComputeSystemsAsync(IDeveloperId? developerId)
+    public async Task<IEnumerable<IComputeSystem>> GetComputeSystemsAsync(IDeveloperId? developerId)

```

Comment:
Should IDeveloperId? be not nullable here? Or we can ask for Compute Systems without it (prompt later?)

Reply:
Good catch. No longer needed; will remove.